dpl.txt

```diff
@@ -23,7 +23,7 @@
 0.550	-0.6139	-0.0003	-0.6136
 0.575	-0.6146	-0.0003	-0.6143
 0.600	-0.6153	-0.0003	-0.6150
-0.625	-0.6159	-0.0003	-0.6156
+0.625	-0.6160	-0.0003	-0.6156
```

Comment:
why are these numbers only up to 4th decimal place? Isn't Python numerical accuracy greater?

Reply:
I think this is just the [rounding convention](https://github.com/jonescompneurolab/hnn-core/blob/6fcc09947e50eca9346c336c02775ba7c527f54b/hnn_core/dipole.py#L532) we've always used when writing to dipole .txt files.

Reply:
right, but our precision for comparison is apparently [higher](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html) than this. Something to think about in the future. Also, yet another reason to move away from text files. The more precision you add, the more bulky it becomes. Because each character is stored as an ASCII rather than the whole number as a floating point.

Reply:
True. What would be a better format?

Reply:
npy / hdf5 / mat ... 

see this for example: https://github.com/bids-standard/bep021/issues/1#issue-712046763